models/Issue.ts

```diff
@@ -20,23 +22,33 @@ export class Issue {
 // tslint:disable-next-line: no-empty
     constructor() { }
     /**
-     * getIssue
+     * Search Issues
      */
-    public async getIssue(id: string, context: SlashCommandContext, read: IRead, http: IHttp, persis: IPersistence): Promise<IIssue> {
+    public async searchIssues(searchText: string,
+                              scope: string,
+                              context: SlashCommandContext,
+                              read: IRead,
+                              http: IHttp,
+                              persis: IPersistence): Promise<Array<IIssue>> {
         const url = await read.getEnvironmentReader().getSettings().getById('url');
         const persistence = new AppPersistence(persis, read.getPersistenceReader());
         const token = await persistence.getAuthToken(context.getSender());
         if (!token) {
             throw new Error('No Valid token found');
         }
         try {
-            const response = await http.get(`${url.value}/projects/27660/issues/${id}`, {
+            const response = await http.get(`${url.value}/issues`, {
                 headers: {
                     'PRIVATE-TOKEN': token,
                 },
+                params: {
+                    search: searchText,
+                    in: 'title',
+                    scope: scope || 'created-by-me',
```

Comment:
In the help function (see above), there are two options: `created-by-me` and `assigned-to-me`. Where is the latter handled? What if I want to search in *all* issues? I expected that without specifying an option, I would just search all.

Reply:
`created-by-me` is default and the `assigned-to-me` comes from the slash command when the user explicitly mention it.

Reply:
Search all issues can be handled by `scope=all` which can lead to lot of issues posted in the channel as each issue will posted as a message in the channel. For now i will leave it simple.

Reply:
Then it should be included in the help function